packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -368,7 +358,8 @@
 
     &:focus {
       background-color: #e4e4e4;
-      outline: none;
+      outline: 2px inset #000;
```

Comment:
Should this have the `dashed` style as well?

Reply:
For consistency's sake, yes.

Reply:
Changed e4930e6

Reply:
@saket2403 Are we able to get the dash border with inset? This looks more like a dash divider than a border around the Today button. If we can't do that with the `outline` style, perhaps try using an inset `box-shadow`?

@jeremyfuksa Any suggestions?

![Screen Shot 2020-03-04 at 12 09 03 PM](https://user-images.githubusercontent.com/22618655/75909373-9aa23c80-5e11-11ea-8836-0351b8abd84f.png)

Reply:
Looks like you can't combine an outline style of `inset` and `dashed`. I added `outline-offset: -3px` and that seems to make an inset dash border. An offset of 2px seems not enough but you can test it out.

![Screen Shot 2020-03-04 at 12 30 30 PM](https://user-images.githubusercontent.com/22618655/75910722-f7065b80-5e13-11ea-8760-043102fe09b8.png)